internal/pkg/api/repository/testresult/mongo.go

```diff
@@ -245,3 +245,15 @@ func composeQueryAndOpts(filter Filter) (bson.M, *options.FindOptions) {
 
 	return query, opts
 }
+
+// DeleteByTest deletes execution results by test
+func (r *MongoRepository) DeleteByTest(ctx context.Context, testName string) (err error) {
+	_, err = r.Coll.DeleteMany(ctx, bson.M{"testsuite.name": testName})
```

Comment:
testsuite? shouldn't it be test ?

Reply:
I guess, naming was wrong for test suite execution methods. But yes, the idea to remove test suite executions when we delete test suite